### PR TITLE
Include backtrace i/o as a crash reporter backend option

### PR DESCRIPTION
**Pull request by philfreo**
Note: I have no affiliation with this company - just thought this would be helpful for people looking for a way to get started quickly (it was for me).

### CODE REVIEW

**Comment by felixrieseberg** [comment]
Makes sense to me. We're happy with Backtrace and I think sharing that option is valuable.

File: docs/api/crash-reporter.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ following projects:
 * [socorro](https://github.com/mozilla/socorro)
 * [mini-breakpad-server](https://github.com/electron/mini-breakpad-server)
 
+Or use a 3rd party hosted solution:
+
+* [Backtrace I/O](http://backtrace.io/electron/)
```
Comment (poiru) [inline docs/api/crash-reporter.md]: Can you switch to `https://`?

Comment (philfreo) [reply docs/api/crash-reporter.md]: All done!

**Comment by abelmathew** [comment]
appreciate the shout-out @philfreo , bump for @MarshallOfSound

**Pull request merged**